### ISSUE TITLE
AST Section nodes not grouping correctly

### ISSUE THREAD

**Issue by jameschensmith** [sivukhin/godjot#9]
#### Summary

With `v1.0.4`, given the following example ([playground link](https://djot.net/playground/?text=%23+Test%0A%0A-+List+item&mode=html&sourcepos=false)):

```
echo -n "# Test\n\n- List item" | go run github.com/sivukhin/godjot@v1.0.4
```

I expect the following output:

```html
<section id="Test">
<h1>Test</h1>
<ul>
<li>
List item
</li>
</ul>
</section>
```

But instead I get the following output:

```html
<section id="Test">
<h1>Test</h1>
</section>
<ul>
<li>
List item
</li>
</ul>
```

I'm not sure if it's just lists. Block quotes seem to work fine.

(Also, thanks so much for your work on this package ❤️ )

**Comment by sivukhin** [comment]
@jameschensmith, happy to hear that someone besides me is using `godjot`!

I fixed your issue and published `v1.0.5` tag which should not have described bug.

**Comment by jameschensmith** [comment]
Of course! Glad to be using it ☺️ 

Awesome! Just pulled it down, and it does look like it's now working 😁 Thank you!
